docs/resources/extended-query-tags.md

```diff
@@ -0,0 +1,214 @@
+| GET    | /extendedquerytags           | Json Array  | [List all supported extended query tags](#list-all-supported-extended-query-tags) |
+| GET    | /extendedquerytags/{tagPath} | Json Object | [Detail an extended query tag's metadata](#get-an-extended-query-tag) |
+| DELETE | /extendedquerytags/{tagPath} |             | [Remove support for specified extended query tag](#remove-an-extended-query-tag) |
+
+### Object Model
+
+| Field          | Type   | Description                                                  |
+| -------------- | ------ | ------------------------------------------------------------ |
+| Path           | string | Path of tag, normally composed of group id and element id. E.g. PatientId (0010,0020) has path 00100020. |
+| VR             | string | Value representation of this tag.                            |
+| PrivateCreator | string | Identification code of the implementer of this private tag. Only set when the tag is a private tag. |
+| Level          | string | Represents the hierarchy at which this tag is relevant.      |
+| Status         | string | Current state this tag is in. Not set when making a request to create a tag. |
+
+### Level
+
+| Level    | Description                                      |
+| -------- | ------------------------------------------------ |
+| Instance | Tag is relevant at the instance level of detail. |
+| Series   | Tag is relevant at the series level of detail.   |
+| Study    | Tag is relevant at the study level of detail.    |
+
+### Status
+
+| Status   | Description                                      |
+| -------- | ------------------------------------------------ |
+| Adding   | The tag is being added to the supported set.     |
+| Ready    | The tag has been added to the supported set.     |
+| Deleting | The tag is being removed from the supported set. |
+
+### Add extended query tags 
+
+### **Route**: /extendedquerytags
+
+**Request Body:**
+
+```
+[
+	{
+		"Path":"04011001", //Private tag
+		"VR":"SS",
+		"PrivateCreator":"MicrosoftPC",
+		"Level":"Instance"
+	},
+	{
+		"Path":"00081090", //Standard tag
+		"VR":"LO",
+		"Level":"Series"
+	},
+	{
+		"Path":"00100040", //Standard tag
+		"VR":"CS",
+		"Level":"Study"
+	},
+]
+```
+
+**Response:**
+
+```
+{}
+```
+
+### **Response status codes:**
+
+| Code              | Description                                          |
+| ----------------- | ---------------------------------------------------- |
+| 202 (Accepted)    | Extended query tag(s) have been successfully stored. |
+| 400 (Bad Request) | Requested extended query path has invalid data.      |
+
+### List all supported extended query tags
+
+**Route:** /extendedquerytags
+
+**Response:**
+
+```
+[
+	{
+		"Path":"04011001", //Private tag
+		"VR":"SS",
+		"PrivateCreator":"MicrosoftPC",
+		"Level":"Instance",
+		"Status":"Added"
+	},
+	{
+		"Path":"00081090", //Standard tag
```

Comment:
use keyword ManufacturerModelName instead of path to demonstrate we support keyword as well?